apis/python/src/tiledbsoma/io/ingest.py

```diff
@@ -1615,7 +1614,7 @@ def update_matrix(
     intended shape of written contents of the array match those of the existing
     data. The intended use-case is to replace updated numerical values.
 
-    Example:
+    Example::
```

Comment:
Sphinx grossly misrenders this unless we have the double colon. Slightly uglier to have a double colon for help(foo) in Python; much uglier to not have the double colon for Sphix.